bokjido/src/main/java/com/projectbusan/bokjido/dto/FacilityReviewDTO.java

```diff
@@ -0,0 +1,68 @@
+package com.projectbusan.bokjido.dto;
+
+import com.projectbusan.bokjido.entity.Facility;
+import com.projectbusan.bokjido.entity.FacilityReview;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Entity;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Schema (description = "시설 리뷰 DTO")
+public class FacilityReviewDTO {
+    @Data
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @Builder
+    public static class FacilityReviewRequestDTO {
+        @Schema(description = "시설 아이디", example = "1")
+        private Long facilityId;
+
+        @Schema(description = "서비스 아이디", example = "1")
+        private Long benefitId;
+
+        @Schema(description = "청결도 별점", example = "5")
+        private int clean;
+
+        @Schema(description = "친절도 별점", example = "5")
+        private int kindness;
+
+        @Schema(description = "주차장 별점", example = "5")
+        private int parking;
```

Comment:
FacilityReviewRequestDTO 클래스는 FacilityReview 엔티티에서 null을 허용하지 않는 조건과 RequestDto로 활용되므로, 아래와 같은 방법으로 수정이 필요해보입니다!
```suggestion
        // clean 예시
        @Schema(description = "청결도 별점 (1~5)", example = "5")
        @Min(value = 1, message = "별점은 최소 1 이상이어야 합니다.")
        @Max(value = 5, message = "별점은 최대 5까지 허용됩니다.")
        private Integer clean;
```

Reply:
조언해주신 코드로 수정 완료했습니다